ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -722,7 +759,23 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
     // The output is only meaningful once view.bounds is non-zero i.e. a layout pass has occured.
     private func sheetFrame(offset: CGFloat) -> CGRect {
         let availableWidth: CGFloat = view.bounds.width
-        let sheetWidth = shouldAlwaysFillWidth ? availableWidth : min(Constants.maxSheetWidth, availableWidth)
+        let maxWidth = min(Constants.maxSheetWidth, availableWidth)
+
+        /// Width will the fill the screen if should always fill width
```

Comment:
These should be // comments since they are inside a function they probably won't be showing as Xcode documentation.